table-api/src/main/java/io/deephaven/api/agg/key/KeyPct.java

```diff
@@ -22,8 +24,10 @@ public boolean averageMedian() {
         return false;
     }
 
-    public Pct withAverage() {
-        return ImmutablePct.builder().percentile(percentile()).pair(pair()).averageMedian(true)
+    public KeyPct withAverage() {
```

Comment:
Do we want these? I don't care either. Maybe better to have a standard approach,